fdbcli/BulkDumpCommand.actor.cpp

```diff
@@ -148,10 +167,10 @@ ACTOR Future<UID> bulkDumpCommandActor(Reference<IClusterConnectionRecord> clust
 
 CommandFactory bulkDumpFactory(
     "bulkdump",
-    CommandHelp("bulkdump [mode|local|clear|status] [ARGs]",
+    CommandHelp("bulkdump [mode|local|blobstore|clear|status] [ARGs]",
                 "bulkdump commands",
                 "To set bulkdump mode: `bulkdump mode [on|off]'\n"
-                "To dump a range to local path in SST files: `bulkdump local <BeginKey> <EndKey> dumpFolder\n"
-                "To clear current bulkdump job: `bulkdump clear\n"
-                "To get completed bulkdump ranges: `bulkdump status <BeginKey> <EndKey>\n"));
+                "To dump a range to SST files: `bulkdump [local|blobstore] <BeginKey> <EndKey> dumpFolder`\n"
```

Comment:
Yeah, in follow-up, can this be 'To dump a range to SST files: `bulkdump create <BeginKey> <EndKey> dumpFolderURL` and then we switch off the URL whether we do blobstore or filesystem?

Reply:
I agree. We can do this in a separate PR later when the UI is finalized.